Libs/Core/ctkUtils.cpp

```diff
@@ -21,7 +21,11 @@
 // Qt includes
 #include <QDebug>
 #include <QDir>
+#if QT_VERSION >= 0x060000
```

Comment:
Please use `QT_VERSION_CHECK(6, 0, 0)` here and everywhere else. It is more readable and more consistent with how Qt versions checks are done now.